python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -1275,6 +1275,16 @@ def _legacy_process_configs(self, config_data, entity_type, project_id, all_acti
             config_path, config_entity = config_data
             commands = []
 
+            # We don't need or want the descriptor object to be sent to the client.
+            # Since we're done with this config for this invokation, we can just
+            # delete it from the entity dict.
+            del config_entity["descriptor"]
```

Comment:
That's fair. I've addressed this in a different branch where I'm currently working.